src/assets/js/pages/NewEventPage/index.js

```diff
@@ -0,0 +1,86 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import {showLoading, hideLoading} from 'react-redux-loading-bar';
+import createValidator from './validate';
+import {registerNewEvent} from '~/data/Api'
+import NewEventForm from './components/NewEventForm';
+import {UncontrolledAlert} from 'reactstrap';
+
+class NewEventPage extends React.Component {
+  static propTypes = {
+    showLoading: PropTypes.func.isRequired,
+    hideLoading: PropTypes.func.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      error: null,
+      event: null
+    }
+
+    this.createNewEvent = this.createNewEvent.bind(this);
+  }
+
+  createNewEvent(event) {
+    registerNewEvent(event)
+      .then((res) => {
+        console.log("res");
+        this.setState({
+          error: null,
+          event: res
+        });
```

Comment:
Maybe better to redirect to the new event page rather than just show an alert

Reply:
Also show an alert on the new event page. There is a redux action for showing adding an alert to an alert page (so you can add it before you even redirect there)
https://github.com/UCSDTESC/Check-in/blob/master/src/assets/js/pages/EventPage/actions/index.js#L3